### PR TITLE
add end-2-end test to the build

### PR DESCRIPTION
**Pull request by joshuawilson**
@kbsingh we need you help since this needs the login/pw

### CODE REVIEW

**Comment by kbsingh** [comment]
Does the work we did last week now help get this unblocked ?

**Comment by joshuawilson** [comment]
[test]

**Comment by joshuawilson** [comment]
[test]

**Comment by joshuawilson** [comment]
[test]

**Comment by hectorj2f** [comment]
@joshuawilson @ldimaggi I can take a look to this code tomorrow morning or tonight. I am on PTO today (it's a public holidays in Spain).

**Comment by joshuawilson** [comment]
@hectorj2f if you can get this to work, that would be great.
@ldimaggi @kbsingh can you take a look at this and see if you can understand why it is failing.

File: cico_run_EE_tests.sh [hunk 1]
```diff
@@ -29,33 +29,11 @@ service docker start
 cp /tmp/jenkins-env .
 docker build -t fabric8-ui-builder -f Dockerfile.builder .
 # User root is required to run webdriver-manager update. This shouldn't be a problem for CI containers
-mkdir -p dist && docker run --detach=true --name=fabric8-ui-builder --user=root --cap-add=SYS_ADMIN -e EE_TEST_USERNAME=$EE_TEST_USERNAME -e EE_TEST_PASSWORD=$EE_TEST_PASSWORD -e "API_URL=http://api.prod-preview.openshift.io/api/" -e "CI=true" -t -v $(pwd)/dist:/dist:Z fabric8-ui-builder
+mkdir -p dist && docker run --detach=true --name=fabric8-ui-builder --user=root --cap-add=SYS_ADMIN -e EE_TEST_USERNAME=$EE_TEST_USERNAME -e EE_TEST_PASSWORD=$EE_TEST_PASSWORD -e "API_URL=http://api.prod.openshift.io/api/" -e "CI=true" -t -v $(pwd)/dist:/dist:Z fabric8-ui-builder
```
Comment (hectorj2f) [inline cico_run_EE_tests.sh]: I believe`http://api.prod.openshift.io/api/` is prod-preview

Comment (hectorj2f) [inline cico_run_EE_tests.sh]: This code doesn work locally... it gives me the following error:

```
    Failed: unknown error: unhandled inspector error: {"code":-32000,"message":"Cannot navigate to invalid URL"}
      (Session info: chrome=60.0.3112.90)
      (Driver info: chromedriver=2.24.417424 (c5c5ea873213ee72e3d0929b47482681555340c3),platform=Linux 4.12.0-0.rc3.git3.2.fc27.x86_64 x86_64)

```

**Pull request closed without merging**